stable/datadog/templates/daemonset.yaml

```diff
@@ -44,7 +48,52 @@ spec:
       {{- if .Values.daemonset.priorityClassName }}
       priorityClassName: {{ .Values.daemonset.priorityClassName }}
       {{- end }}
+      {{- if $.Values.datadog.system_probe.enabled }}
+      initContainers:
+      - name: seccomp-setup
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+        command:
+        - cp
+        - /etc/config/system-probe-seccomp.json
+        - /host/var/lib/kubelet/seccomp/system-probe
+        volumeMounts:
+        - name: datadog-agent-security
+          mountPath: /etc/config
+        - name: seccomp-root
+          mountPath: /host/var/lib/kubelet/seccomp
+        resources:
```

Comment:
You can leave out the `resources` field because it's inferred from the (larger) requests/limits of the non-init containers

https://kubernetes.io/docs/concepts/workloads/pods/init-containers/#resources